tomviz/AddPythonTransformReaction.cxx

```diff
@@ -219,6 +219,35 @@ OperatorPython* AddPythonTransformReaction::addExpression(DataSource* source)
           source->addOperator(op);
       }
   }
+  else if (scriptLabel == "Sobel Filter") //UI for Sobel Filter
+  {
+      QDialog dialog(pqCoreUtilities::mainWidget());
+      QHBoxLayout *layout = new QHBoxLayout;
+      QLabel *label = new QLabel("Axis:");
+      layout->addWidget(label);
+      QSpinBox *axis = new QSpinBox;
+      axis->setRange(0, 2);
+      axis->setValue(0);
```

Comment:
If this represents axis, we should hide the numbers from the user.  They probably don't care that 0 is X, 1 is Y and 2 is Z.  We should probably make this a QComboBox with the items "X", "Y" and "Z" in the drop down.